core/src/test/java/org/apache/accumulo/core/clientImpl/TabletLocatorImplTest.java

```diff
@@ -1281,12 +1281,8 @@ public void testAccumulo1248() {
     setLocation(tservers, "tserver2", MTE, ke1, "L1", "I1");
     setLocation(tservers, "tserver2", MTE, ke1, "L2", "I2");
 
-    try {
-      metaCache.locateTablet(context, new Text("a"), false, false);
-      fail();
-    } catch (Exception e) {
-
-    }
+    assertThrows(Exception.class,
+        () -> metaCache.locateTablet(context, new Text("a"), false, false));
```

Comment:
It'd be great if this code was more specific about which exception it was expecting.

Reply:
Addressed in 2c61494